src/contracts/handlers/ERC20Handler.sol

```diff
@@ -42,7 +42,7 @@ contract ERC20Handler is IHandler, ERCHandlerHelpers, ERC20Safe {
         bytes32 resourceID,
         address depositor,
         bytes calldata data
-    ) external override onlyBridge returns (bytes memory) {
+    ) external override onlyBridgeRouterOrExecutor returns (bytes memory) {
```

Comment:
I would keep most of these as onlyBridge because Router or Executor don't even have a way to call them.